src/reasoning/SWRL/swrl.pl

```diff
@@ -301,3 +339,40 @@
 swrl_nums([X|Xs],[Y|Ys],Vars) :-
   swrl_atom_number(X,Y,Vars),
   swrl_nums(Xs,Ys,Vars).
+
+%% swrlx storage
+:- dynamic swrlx_individual_map/2.
+
+swrlx_add_individual(Pattern,Individual) :-
+    (   swrlx_individual_map(Pattern, Individual)
```

Comment:
I am a bit confused about what this pattern is, and why there is a 1-to-1 mapping between pattern and individual.
Could you add a comment/example?

Reply:
Maybe you could copy some of the pull request comment into the source code, this would help reading the code